lib/counter_culture.rb

```diff
@@ -13,7 +13,13 @@ module ActiveRecord
 
     module ClassMethods
       # this holds all configuration data
-      attr_reader :after_commit_counter_cache
+      def after_commit_counter_cache
+        config = @after_commit_counter_cache
+        if !config && superclass.respond_to?(:after_commit_counter_cache)
+          config = superclass.after_commit_counter_cache
+        end
+        config
+      end
```

Comment:
It actually looks like it might be a use case for the much-maligned `@@` variable, though who knows what other horrible side effects that might have...